### PR TITLE
[BE-139] System report: osx-xcode-13.1.x

### PR DESCRIPTION
**Pull request by AndrasHacks**
DO NOT MERGE UNTIL RELEASED TO PRODUCTION!!

Attributes checked and cross-validated: https://bitrise.atlassian.net/wiki/spaces/INFRA/pages/2429059073/Checklist+for+BE-Sprint+4+2022-07-25+-+2022-08-05

### CODE REVIEW

**Review by fehersanyi-bitrise — APPROVED** [review]
ql

File: system_reports/MACOS/INTEL/osx-xcode-13.1.x.log [hunk 1]
```diff
@@ -2130,7 +2185,7 @@ faraday-rack (1.0.0)
 faraday-retry (1.0.3)
 faraday_middleware (1.2.0)
 fastimage (2.2.6)
-fastlane (2.207.0, 2.206.2)
+fastlane (2.208.0, 2.207.0, 2.206.2)
```
Comment (lpusok) [inline system_reports/MACOS/INTEL/osx-xcode-13.1.x.log:2188]: There are more fastlane versions installed. Just curios, did we deliberately installed new version or this is just an additional dependency from somewhere?

**Review by lpusok — APPROVED** [review]

File: system_reports/MACOS/INTEL/osx-xcode-13.1.x.log (hunk 1 above)
Comment (AndrasHacks) [reply system_reports/MACOS/INTEL/osx-xcode-13.1.x.log:2188]: @fehersanyi-bitrise , @pgyula - can you please answer this?

Comment (fehersanyi-bitrise) [reply system_reports/MACOS/INTEL/osx-xcode-13.1.x.log:2188]: added dependencies from some magical place

**Review by pgyula — APPROVED** [review]

**Pull request merged**